core/sail/shacl/src/main/java/org/eclipse/rdf4j/sail/shacl/planNodes/GroupByCount.java

```diff
@@ -74,7 +75,8 @@ private void calculateNext() {
 
 				}
 
-				List<Value> line = Arrays.asList(subject, SimpleValueFactory.getInstance().createLiteral(count));
+				List<Value> line = new ArrayList<>(
+						Arrays.asList(subject, SimpleValueFactory.getInstance().createLiteral(count)));
```

Comment:
Why are you wrapping this in an Arraylist?

Reply:
Arrays.asList is immutable and I need mutability.

Reply:
I've added some comments.